yarn-project/bb-prover/src/bb/execute.ts

```diff
@@ -144,8 +151,7 @@ export async function generateKeyForNoirCircuit(
         result = await executeBB(pathToBB, `vk_as_fields`, asFieldsArgs, log);
       }
       const duration = timer.ms();
-      // Cleanup the bytecode file
```

Comment:
I think it's better if the inputs (and outputs) are not cleaned up here. It's already done by the caller.